addons/account/views/account_move_views.xml

```diff
@@ -970,29 +970,50 @@
                                 <label for="journal_id"
                                        groups="account.group_account_readonly"
                                        invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"/>
+                                <label for="currency_id"
```

Comment:
Here are a few suggestions to simplify the view (and fix some parts)
* manage the labels for the `journal_div` using only groups to be sure to have a label if and only if the div is shown (don't know why we would rely on the context, the condition on the move type is on `currency_div`)
* remove the `<t/>
* remove useless classes on divs
* use the whitespace style on the whole `currency_div`
* inline and fix indentation of `currency_conversion_div` elements


```diff
diff --git a/addons/account/views/account_move_views.xml b/addons/account/views/account_move_views.xml
index 2afbcff6fba5..44722f3aad58 100644
--- a/addons/account/views/account_move_views.xml
+++ b/addons/account/views/account_move_views.xml
@@ -967,50 +967,36 @@
                                            options="{'no_quick_create':True}" readonly="state in ['cancel', 'posted']"/>
                                 </div>
                                 <field name="delivery_date" invisible="not show_delivery_date" readonly="state != 'draft'"/>
-                                <label for="journal_id"
-                                       groups="account.group_account_readonly"
-                                       invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"/>
-                                <label for="currency_id"
-                                       groups="!account.group_account_readonly,base.group_multi_currency"
-                                       invisible="move_type == 'entry'"/>
+                                <label for="journal_id" groups="account.group_account_readonly"/>
+                                <label for="currency_id" groups="!account.group_account_readonly,base.group_multi_currency"/>
                                 <div name="journal_div"
                                      class="d-flex"
-                                     invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'">
-                                    <t groups="account.group_account_readonly">
-                                        <field name="journal_id"
-                                               options="{'no_create': True, 'no_open': True}"
-                                               readonly="posted_before"/>
-                                        <span class="oe_inline o_form_label mx-3"
-                                              groups="base.group_multi_currency"
-                                              invisible="move_type == 'entry'"> in </span>
-                                    </t>
+                                     groups="account.group_account_readonly,base.group_multi_currency">
+                                    <field name="journal_id"
+                                           groups="account.group_account_readonly"
+                                           options="{'no_create': True, 'no_open': True}"
+                                           readonly="posted_before"/>
                                     <div name="currency_div"
                                          groups="base.group_multi_currency"
+                                         style="white-space: pre;"
                                          invisible="move_type == 'entry'">
+                                        <span class="o_form_label"
+                                              groups="account.group_account_readonly"
+                                              invisible="move_type == 'entry'"> in </span>
                                         <field name="currency_id"
-                                               nolabel="1"
                                                readonly="state != 'draft'"
+                                               class="oe_inline"
                                                options="{'no_create': True}"
                                                context="{'search_default_active': 1, 'search_default_inactive': 1}"/>
                                         <div name="currency_conversion_div"
-                                             class="d-flex text-muted float-start"
-                                             style="white-space: pre;"
+                                             class="d-flex text-muted"
                                              invisible="currency_id == company_currency_id">
-                                                <span class="oe_inline o_form_label">1 </span>
-                                                <field name="company_currency_id"
-                                                       readonly="True"
-                                                       options="{'no_open': True}"
-                                                       />
-                                                <span class="oe_inline o_form_label"> = </span>
-                                                <field name="invoice_currency_rate"
-                                                       widget="float"
-                                                       digits="[12,6]"
-                                                       readonly="True"/>
-                                                <span class="oe_inline o_form_label"> </span>
-                                                <field name="currency_id"
-                                                       readonly="True"
-                                                       options="{'no_open': True}"
-                                                       />
+                                            <span class="o_form_label">1 </span>
+                                            <field name="company_currency_id" readonly="True" options="{'no_open': True}"/>
+                                            <span class="o_form_label"> = </span>
+                                            <field name="invoice_currency_rate" digits="[12,6]" readonly="True"/>
+                                            <span class="o_form_label"> </span>
+                                            <field name="currency_id" readonly="True" options="{'no_open': True}"/>
                                         </div>
                                     </div>
                                 </div>
```

Reply:
Thanks. I made some small changes to make it look nicer (imo)
- Align the "in" and the "1" of the conversion string (c.f. first pic. below)
- Get rid of the weird space in the conversion string (c.f. first pic. below; `float-start`)
- Add an explicit line break between the currency_id and the conversion string
- Make conversion string look normal on small screens (c.f. second pic. below; span classes)
![2024-02-29_11-21](https://github.com/odoo/odoo/assets/136557140/1c1feb76-f512-49ec-b3c0-ca3214a07490)
![2024-02-29_11-55_small_screen_issue](https://github.com/odoo/odoo/assets/136557140/c02f0810-fb2d-42ea-b473-f4ec0346a16b)

```diff
diff --git a/addons/account/views/account_move_views.xml b/addons/account/views/account_move_views.xml
index 44722f3aad58..e0d22ec17218 100644
--- a/addons/account/views/account_move_views.xml
+++ b/addons/account/views/account_move_views.xml
@@ -974,6 +974,7 @@
                                      groups="account.group_account_readonly,base.group_multi_currency">
                                     <field name="journal_id"
                                            groups="account.group_account_readonly"
+                                           class="mr4"
                                            options="{'no_create': True, 'no_open': True}"
                                            readonly="posted_before"/>
                                     <div name="currency_div"
@@ -982,20 +983,21 @@
                                          invisible="move_type == 'entry'">
                                         <span class="o_form_label"
                                               groups="account.group_account_readonly"
-                                              invisible="move_type == 'entry'"> in </span>
+                                              invisible="move_type == 'entry'">in </span>
                                         <field name="currency_id"
                                                readonly="state != 'draft'"
                                                class="oe_inline"
                                                options="{'no_create': True}"
                                                context="{'search_default_active': 1, 'search_default_inactive': 1}"/>
+                                        <div class="break"/>
                                         <div name="currency_conversion_div"
-                                             class="d-flex text-muted"
+                                             class="d-flex text-muted float-start"
                                              invisible="currency_id == company_currency_id">
-                                            <span class="o_form_label">1 </span>
+                                            <span>1 </span>
                                             <field name="company_currency_id" readonly="True" options="{'no_open': True}"/>
-                                            <span class="o_form_label"> = </span>
+                                            <span> = </span>
                                             <field name="invoice_currency_rate" digits="[12,6]" readonly="True"/>
-                                            <span class="o_form_label"> </span>
+                                            <span> </span>
                                             <field name="currency_id" readonly="True" options="{'no_open': True}"/>
                                         </div>
                                     </div>
```